projects/plugins/jetpack/modules/search/instant-search/components/product-ratings.jsx

```diff
@@ -0,0 +1,51 @@
+/** @jsx h */
+/**
+ * External dependencies
+ */
+import { _n, sprintf } from '@wordpress/i18n';
+import { h } from 'preact';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from './gridicon';
+
+/**
+ * Style dependencies
+ */
+import './product-ratings.scss';
+
+/**
+ * Renders a hook-based component for displaying product ratings.
+ *
+ * @param {object} props - Component properties.
+ * @param {number} props.count - Number of ratings.
+ * @param {number} props.rating - Average rating out of five.
+ * @returns {object} Product rating component.
+ */
+export default function ProductRatings( { rating = 0, count = 0 } ) {
+	return (
+		<div className="jetpack-instant-search__search-result-product-rating">
+			<span aria-hidden className="jetpack-instant-search__search-result-product-rating-stars">
+				{ Array( 5 )
+					.fill( <Gridicon size={ 16 } icon="star-outline" /> )
+					.fill( <Gridicon size={ 16 } icon="star" />, 0, rating ) }
+			</span>{ ' ' }
+			<span aria-hidden className="jetpack-instant-search__search-result-product-rating-count">
+				{ count }
+			</span>
+			<span className="screen-reader-text">
+				{ sprintf(
+					_n(
+						'Average rating of %s out of 5 from %s rating.',
```

Comment:
Shall we do "from %s review" / "from %s reviews" instead? That's how the Woo frontend refers to it:

<img width="559" alt="Screen Shot 2021-01-21 at 14 42 25" src="https://user-images.githubusercontent.com/17325/105268354-ef4fd100-5bf6-11eb-99f8-95c3d0a0441b.png">

Reply:
👍